csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -118,7 +118,7 @@ static JsonFormatter()
             {
                 if (CommonRepresentations[i] == "")
                 {
-                    CommonRepresentations[i] = ((char) i).ToString();
+                    CommonRepresentations[i] = ((char)i).ToString();
```

Comment:
Please could you revert all of the white-space only changes? It makes it much harder to find all the *actual* changes. We can then do a whites-space only reformat if absolutely required, although I'd prefer to just leave it as it is.